app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<header>
+  <%= render partial: 'search_form' %>
+</header>
```

Comment:
Since you include the search form on every page, it might make sense to put it in `application.html.erb`.